woocommerce-customizer.php

```diff
@@ -334,6 +337,41 @@ public static function render_outdated_wc_version_notice() {
 	}
 
 
+	/**
+	 * Renders a button which opens the media library model
+	 *
+	 * Also renders an image field to display the currently selected image
+	 * and a hidden field used to store the selected image URL
+	 *
+	 * @since 2.6.1
+	 */
+	public function render_media_library_field() {
+
+		/*
+		 * Get the placeholder image URL directly from settings so the admin form
+		 * shows the correct image immediately after saving.
+		 */
+		$placeholder_url = $this->settings->customizations["woocommerce_placeholder_img_src"];
+
+		?>
+		<table class="form-table">
+			<tbody><tr valign="top">
+				<th scope="row" class="titledesc">
+					<label for="woocommerce_placeholder_img_src"><?php _e( 'Placeholder Image' ); ?></label>
```

Comment:
We tend to escape translations given we're not often working directly with translators we trust. Also need a textdomain here for localization. So we'd use `esc_html_e( 'Placeholder image', 'woocommerce-customizer' );`